Organiser.Core/Organiser.Core/Controllers/BugsController.cs

```diff
@@ -0,0 +1,120 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Organiser.Core.Models.ViewModels.BugsViewModels;
+using Organiser.Cores.Context;
+using Organiser.Cores.Entities;
+using Organiser.Cores.Models.Enums;
+using Organiser.Cores.Services;
+using System;
+
+namespace Organiser.Cores.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    [Authorize]
+    public class BugsController : ControllerBase
+    {
```

Comment:
btw I suggest you to get familiar with minimal API. it is a modern way of defining endpoints and actually it is a standard nowadays. Controller's are no longer fashionable 😛